crates/next-api/src/module_graph.rs

```diff
@@ -316,12 +355,16 @@ impl ServerActionsGraph {
     pub async fn new_with_entries(
         graph: ResolvedVc<SingleModuleGraph>,
         is_single_page: bool,
+        graph_idx: u32,
+        binding_usage: Option<ResolvedVc<BindingUsageInfo>>,
     ) -> Result<Vc<Self>> {
         let mapped = map_server_actions(*graph);
 
         Ok(ServerActionsGraph {
             is_single_page,
             graph,
+            graph_idx,
+            binding_usage,
```

Comment:
That and the `read(...)` call sounds a little bit like you wan to have `.with_binding_usage()` here:


```suggestion
            graph.with_binding_usage(graph_idx, binding_usage)
```

Reply:
Probably even higher up in the call stack